### ISSUE TITLE
CSV to JSON is incorrectly tranformed if there is a field that starts with a break line.

### ISSUE THREAD

**Issue by AndrzejKmicic** [mrodrig/json-2-csv#265]
## Background Information
- Module Version: ` `
- Node/Browser Version: ` `

The issue I'm reporting is with:
- [ ] json2csv
- [x] csv2json

I have...
- [x] searched to see if an issue has already been reported.
- [x] verified that my JSON/CSV data is valid (using something like http://jsonlint.com or https://csvlint.io/).
- [x] tried upgrading to the latest version of json-2-csv (since the issue may already be fixed).

## Expected Behavior
CSV to JSON is correctly tranformed if there is a field that starts with a break line.

```
{"id":1,"name":"John Doe","age":29,"city":"New York"}
{"id":2,"name":"Jane Smith","age":34,"city":"\n Los Angeles"}
{"id":3,"name":"Emily Davis","age":22,"city":"Chicago"}
{"id":4,"name":"Michael Brown","age":40,"city":"Houston"}
```

## Actual Behavior
CSV to JSON is incorrectly tranformed if there is a field that starts with a break line.

```
{"id":1,"name":"John Doe","age":29,"city":"New York"}
{"id":2,"name":"Jane Smith","age":"34,\""}
{"id":" Los Angeles\""}
{"id":3,"name":"Emily Davis","age":22,"city":"Chicago"}
{"id":4,"name":"Michael Brown","age":40,"city":"Houston"}
```

## Data Sample 
CSV:
```
id,name,age,city\n1,John Doe,29,New York\n2,Jane Smith,34,"\n Los Angeles"\n3,Emily Davis,22,Chicago\n4,Michael Brown,40,Houston
```

## Code Example
[Stackblitz](https://stackblitz.com/edit/vitejs-vite-s8cqmi?file=package.json,src%2FApp.jsx,src%2FApp.css&terminal=dev)

**Comment by mattrobinson10** [comment]
I'm also experiencing this issue. It looks like it might have previously been fixed here https://github.com/mrodrig/json-2-csv/issues/240, I wonder if a regression has been introduced at some point?

**Comment by mattrobinson10** [comment]
I have checked the [release](https://github.com/mrodrig/json-2-csv/releases/tag/5.0.1) generated from https://github.com/mrodrig/json-2-csv/issues/240 , and the issue is present there as well, so I don't think this issue is a regression after all.

**Comment by mrodrig** [comment]
Thanks for reporting this @AndrzejKmicic and for testing it against the release from #240 @mattrobinson10. Sorry for the delay in addressing this - I had some personal commitments that needed my full attention as a family member wasn't well. I've started looking into this issue and think I have a general idea as to why this is happening. I'll keep you updated when I have a solution. Thanks again for your understanding.

**Comment by mattrobinson10** [comment]
No problem at all @mrodrig, sorry to hear that a family member wasn't well I hope they are feeling better now. If I can help with anything let me know!

**Comment by mrodrig** [comment]
This should be resolved in the latest release (5.5.7). Please let me know if you run into any other issues and thanks again for your help tracking this down. 🙂

**Comment by mattrobinson10** [comment]
Thanks for the super quick fix! 🙌
